src/python/pants/option/global_options.py

```diff
@@ -96,12 +96,11 @@ def register_bootstrap_options(cls, register):
                   'subprocesses that outlive the workdir data (e.g. `./pants server`).')
     register('--pants-config-files', advanced=True, type=list,
```

Comment:
Sure. It seems a bit wasteful to pull from the limited short-flag universe for an advanced option, but we have a few precedents :/